src/middlewared/middlewared/alert/source/scrub_paused.py

```diff
@@ -0,0 +1,21 @@
+import libzfs
+
+from middlewared.alert.base import Alert, AlertLevel, ThreadedAlertSource
+from middlewared.alert.schedule import CrontabSchedule
+
+
+class ScrubPausedAlertSource(ThreadedAlertSource):
+    level = AlertLevel.WARNING
+    title = "Scrub is paused"
+
+    schedule = CrontabSchedule(hour=13)
```

Comment:
Why hour 13?

Reply:
It was 13 when I was testing it :) I'll update it to 3 as soon as py-libzfs update will be available